psique/src/App/test/king/king.js

```diff
@@ -1,8 +1,576 @@
+        <Dialog aria-labelledby="simple-dialog-title" open={modalControl}>
+          <DialogTitle id="simple-dialog-title">Por favor confirme los resultados</DialogTitle>
+          <Divider />
+            <DialogContent>              
+              <DialogContentText>Paciente: {patientName} </DialogContentText>
+              <DialogContentText>ID: {patientId}</DialogContentText>
+              <br></br>
+              <DialogContentText> Fecha: {testDate} </DialogContentText>
+              <DialogContentText >Tiempo (segundos): {time}</DialogContentText>
+              <DialogContentText >Puntaje: {score}</DialogContentText>
+              <br></br>
+              <DialogContent>
+                <Grid container justify="center" spacing={2}>
+                  <Grid item xs={4}> <Link to={'/home'}><Button variant="contained" color="primary">Sí</Button></Link> </Grid>
+                  <Grid item xs={4}> <Button variant="contained" color="secondary" onClick={() => {setModalControl(false)}}>No</Button> </Grid>
+                </Grid>
+              </DialogContent>
+            </DialogContent>
+        </Dialog>    
+      </div>
+    );
+  }
+
+  return (   
+    <div className={classes.reyTestpage}>
+      <br></br>
+      {showResults()}
+      <Container justify="center" align="center" fixed>      
+      <Paper className={classes.paper}>
+        <br></br>
+        <h1 className={classes.classTitle}>Test de Rey</h1>
+        <Divider />
+        <br></br>
+        <Grid container justify="center" spacing={2}>
+          <Grid item>
+            Paciente: {patientName}
+          </Grid>
+          <Grid item>
+            ID: {patientId} 
+          </Grid>
+        </Grid>
+
+        <br></br>
+        <form align="center">
+          <Grid container justify="center" spacing={2}>            
+            <TextField
+              id="date"
+              label="Fecha de la Prueba"
+              type="date"
+              value={testDate}
+              onChange={(e)=>setTestDate(e.target.value)}
+              className={classes.textField}
+              InputLabelProps={{
+                shrink: true,
+              }}
+            />
+          </Grid>
+
+          <br></br>
+          <Grid container justify="center" spacing={2}>
+            <Grid item> <FormLabel> Tiempo: </FormLabel> </Grid>
+            <Grid item> <TextField required label="Minutos" value={minutes} onChange={(e)=>setMinutes(e.target.value)} maxLength="3" inputProps={{ 'aria-label': 'minutes' }} /> </Grid>
+            <Grid item> <TextField required label="Segundos" value={seconds}  onChange={(e)=>setSeconds(e.target.value)} maxLength="2" inputProps={{ 'aria-label': 'seconds' }} /> </Grid>
+          </Grid>
+          <br></br>
+
+          <Grid container justify="center">
+            <TextField  id="copyType" multiline label="Tipo de Copia" onChange={(e)=>setCopyType(e.target.value)} value={copyType} inputProps={{ 'aria-label': 'copy-type' }}/>
```

Comment:
U can use a combobox to limit the possible answers

Reply:
Are you refering to copyType textfield? I leave it open for specialists to write, even if the guide give some pointers, it looks like it more free to write observations of the specific drawn figure of the patient

Reply:
Ohh mai gash, it's ok so v: